Makefile

```diff
@@ -120,12 +120,15 @@ MARIADB_ASSERT      ?= ${MARIADB_KUTTL_DIR}/mariadb_deploy/01-assert.yaml
 endif
 
 # Placement
-PLACEMENT_IMG         ?= quay.io/openstack-k8s-operators/placement-operator-index:latest
-PLACEMENT_REPO        ?= https://github.com/openstack-k8s-operators/placement-operator.git
-PLACEMENT_BRANCH      ?= main
-PLACEMENTAPI          ?= config/samples/placement_v1beta1_placementapi.yaml
-PLACEMENTAPI_CR       ?= ${OPERATOR_BASE_DIR}/placement-operator/${PLACEMENTAPI}
-PLACEMENTAPI_DEPL_IMG ?= unused
+PLACEMENT_IMG           ?= quay.io/openstack-k8s-operators/placement-operator-index:latest
+PLACEMENT_REPO          ?= https://github.com/openstack-k8s-operators/placement-operator.git
+PLACEMENT_BRANCH        ?= main
+PLACEMENTAPI            ?= config/samples/placement_v1beta1_placementapi.yaml
+PLACEMENTAPI_CR         ?= ${OPERATOR_BASE_DIR}/placement-operator/${PLACEMENTAPI}
+PLACEMENTAPI_DEPL_IMG   ?= unused
+PLACEMENT_KUTTL_CONF    ?= ${OPERATOR_BASE_DIR}/placement-operator/kuttl-test.yaml
+PLACEMENT_KUTTL_DIR     ?= ${OPERATOR_BASE_DIR}/placement-operator/test/kuttl/tests
+PLACEMENT_KUTTL_TIMEOUT ?= 180
```

Comment:
We need to add a `PLACEMENT_KUTTL_TEST_DIR` which should have the same value as `PLACEMENT_KUTTL_DIR` if I'm reading correctly from https://github.com/openstack-k8s-operators/placement-operator/blob/c3790f3ec61f7605306f85ced31d2e8dd9f3a047/test/kuttl/common/assert_sample_deployment.yaml#L285C10-L285C34

Reply:
Ugh.... Why do we need a different variable ? I think we should fix that inconsistent naming in placement-operator instead.

Reply:
I've pushed https://github.com/openstack-k8s-operators/placement-operator/pull/24 .

Reply:
fair enough, thanks @kajinamit